libr/cons/dietline.c

```diff
@@ -1255,6 +1256,12 @@ R_API const char *r_line_readline_cb(RLineReadCallback cb, void *user) {
 		__print_prompt ();
 	}
 	r_cons_break_push (NULL, NULL);
+	int mouse_status = cons->mouse;
+	if (I.hud) {
+		r_cons_enable_mouse (true);
```

Comment:
what about r_cons_enable-mouse (I.hid); instead ?

Reply:
Yes, it is better. Changed it.